src/components/nodes/Boolean.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="q-px-md q-pt-md">
+    <q-toggle
+      :label="toggle"
```

Comment:
We could consider using `:label="toggle ? 'yes' : 'no` instead for readibility. Similar to how "text" is more comprehensible than "string" for a paleograph.